<?xml version="1.0"?>
<launch>

    <node pkg="mir_ppt_detection" type="ppt_detector" name="ppt_detector" output="screen">
        <remap from="~points" to="/arm_cam3d/depth_registered/points"/>
    </node>

    <node pkg="mir_ppt_detection" type="ppt_classifier_node" name="ppt_classifier_node" output="screen">
        <remap from="~cavities" to="/ppt_detector/cavities"/>
    </node>
</launch>
